frontend/src/components/Editor.js

```diff
@@ -58,6 +69,10 @@ export default function BasicEditor() {
         "ace_active-line",
         "fullLine"
       );
+      if (location.file !== files[currentFile].fileId) {
```

Comment:
The line will be incorrect right after switching files

Reply:
My intention is that after switching file, the current file is set then if the incoming location returned by the socket is different then we find the file and switch it otherwise, we keep the current file.

Reply:
Hm, I see. I'll fix this.